doc/bp-Command.md

```diff
@@ -0,0 +1,231 @@
+## ``bp`` - The BiblioPixel Project Runner
```

Comment:
This documentation is now in two places - here and command.py - and it's slightly different, even! 

Can we eliminate one of them?

Reply:
Actually, I'm against this entire file even existing, because it means we have this documentation in _three_ places:

* command.py
* this file
* `bp --help`

I have a facility for extracting `bp --help` automatically into a file - we should be using that and not checking this in.

Reply:
Crap... need to update .gitignore. This should not have been comitted. It's supposed to be auto-generated from command.py.
Actually it is and would get overwritten, it just should not have been committed.